src/modules/claim/services/claim.service.ts

```diff
@@ -306,6 +306,10 @@ export class ClaimService {
     }
     const rolesByRevoker = allRolesByRevoker.map((r) => r.namespace);
 
+    if (!rolesByRevoker.length) {
```

Comment:
If rolesRevokers can't be undefined then we can return just roleRevokers

Reply:
this function needs to return claims not roles